padinfo/padinfo.py

```diff
@@ -512,32 +513,80 @@ async def collabscroll(self, ctx, *, query: str):
             await ctx.send(self.makeFailureMsg(err))
 
 
+    async def _parse_ls_query(self, query : str) -> (str, str): 
+        '''Helper function for leaderskill. Takes in a query 
+        and returns the left or right'''
+        if not query: 
+            raise Exception('Query seems to be empty.')
+
+        # i assume neither '"' nor '/' will be used as part of card search
+        # this handles splitting with quotewrapping and /
+        # also comma splitting if there are quotes
+        if '"' in query or '/' in query or ',' in query: 
+            p = r'(?:(?:([^"/,]*))|(?:"([^"/]*)"))\s*(?:/|,)?\s*(?:(?:(?:([^"/,]*))|(?:"([^"/]*)")))?'
```

Comment:
`re.split(pattern, string)` might be a better way to do this.  This regex is kinda hard to parse manually!